parm/namelist.wps

```diff
@@ -24,7 +24,7 @@
   truelat1 = 38.5,
   truelat2 = 38.5,
   stand_lon = -97.5
-  geog_data_path = '',
+  geog_data_path = '/mnt/lfs5/BMC/wrfruc/HRRRv5/geog/',
```

Comment:
We don't need to define line 27 in the workflow.

Reply:
reset to geog_data_path = ''.  WPS (ungrib) does not need to set this path.